packages/page-treasury/src/Overview/ProposalCreate.tsx

```diff
@@ -99,7 +105,7 @@ function Propose({ className }: Props): React.ReactElement<Props> | null {
               isDisabled={!accountId || !hasValue}
               label={t<string>('Submit proposal')}
               onStart={toggleOpen}
-              params={[value, beneficiary, segment]}
+              params={[value, beneficiary, segment, cycle]}
```

Comment:
[Value, Beneficiary, Segment, Cycle]

Reply:
For this suggestion, the variables are declared in lowercases, so these will not work as parameters as they are undefined.

Reply:
Oh right, makes sense then.